apps/website/src/pages/show-and-tell/my-posts/[postId]/index.tsx

```diff
@@ -6,16 +6,18 @@ import { trpc } from "@/utils/trpc";
 
 import { LoginWithTwitchButton } from "@/components/shared/LoginWithTwitchButton";
 import { MessageBox } from "@/components/shared/MessageBox";
+import { PageNavigation } from "@/components/shared/PageNavigation";
 
 import Meta from "@/components/content/Meta";
 import Section from "@/components/content/Section";
 import Heading from "@/components/content/Heading";
 
-import { ShowAndTellNavigation } from "@/components/show-and-tell/ShowAndTellNavigation";
 import { ShowAndTellEntryForm } from "@/components/show-and-tell/ShowAndTellEntryForm";
 
 import showAndTellHeader from "@/assets/show-and-tell/header.png";
 
+import { showAndTellNavItems } from "../..";
```

Comment:
Note sure i like the relative path here. Maybe we can just use the full path or adding back the ShowAndTellNavigation component (basically just wrapping the `PageNavigation`) in its own file and co-locate the items there?